src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -255,6 +256,11 @@ protected void SetReturnsVoid(bool returnsVoid)
             this.flags.ReturnsVoid = returnsVoid;
         }
 
+        protected void SetIsExtensionMethod(bool isExtensionMethod)
```

Comment:
>protected void SetIsExtensionMethod(bool isExtensionMethod) [](start = 8, length = 59)

Please remove the method, this an immutable state of the symbol, it should not be mutated once accessed.